tests/flow-force-reuse/flow-force-reuse-on/test.yaml

```diff
@@ -0,0 +1,13 @@
+pcap: ../test.pcap
+
+args:
+- --set flow.memcap-policy=drop-flow
+- --set flow.prealloc=0
+- --set flow.memcap=4kb
+- --set flow.hash-size=32
+- --set flow.force-reuse=true
+
+checks:
+  - shell:
+      args: grep -q 'flow.get_used \|flow.get_used_eval ' stats.log && printf '1\n' || printf '0\n'
```

Comment:
I prefer using the eve.json based checks

Reply:
This patch is using grep on stats.log, rather than usual eve.json, to test for existence of flow.get_used or flow.get_used_eval rather than a specific number.
Is there a way to do this with eve.json?

Thanks

Reply:
would this work? https://github.com/OISF/suricata-verify/blob/master/README.md?plain=1#L148